Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptVuejsApp/TypeScriptVuejsApp.vstemplate

```diff
@@ -12,6 +12,9 @@
     <NumberOfParentCategoriesToRollUp>1</NumberOfParentCategoriesToRollUp>
     <PromptForSaveOnCreation>true</PromptForSaveOnCreation>
     <TemplateGroupID>Node.js</TemplateGroupID>
+    <LanguageTag>Typescript</LanguageTag>
+    <PlatformTag>Windows</PlatformTag>
```

Comment:
Why Windows?  Seems like this would work anywhere?

Reply:
I agree but there's no option for 'any' on the templates and the others are misleading. Here's the documentation [Built-in tags](https://docs.microsoft.com/en-us/visualstudio/ide/template-tags?view=vs-2019#built-in-tags). The other templates are tagged as Windows as well.

Reply:
The example in the docs makes it look like you can tag multiple platforms.

Reply:
I suppose I can add Linux and MacOS as well as per Node.js support. I'll fix it in second.